siloLapisTests/test/aggregated.spec.ts

```diff
@@ -196,4 +195,12 @@ null	Switzerland	2
     expect(result.status).equals(200);
     expect(result.headers.get('lapis-data-version')).to.match(/\d{10}/);
   });
+
+  it('should return the data with Content-Disposition when asking for download', async () => {
```

Comment:
Should we add this test to a common spec file, since we need it for all other routes? (There are others also like this, dataVersion etc.)

Reply:
I'll do it in a separate, follow-up PR :+1:

Reply:
-> #613